src/Util/VerifyEmailQueryUtility.php

```diff
@@ -23,13 +25,19 @@ class VerifyEmailQueryUtility
 {
     public function getTokenFromQuery(string $uri): string
     {
+        /** @psalm-suppress UndefinedFunction */
+        @trigger_deprecation('symfonycasts/verify-email-bundle', '1.17.0', 'This method is deprecated and will be removed in 2.0.');
```

Comment:
Curious that this needs the pslam-suppress... since the library does require the deprecations-contract

Reply:
Ya, I don't get it either - locally I don't get the error. But on CI, we do. I think it has something to do with psalm not picking up the autoload. 

![image](https://github.com/SymfonyCasts/verify-email-bundle/assets/40327885/a5931696-cedf-410e-9484-ff475ba094d6)

Reply:
I'll check into this outside of this PR.